logger.py

```diff
@@ -0,0 +1,73 @@
+import time
+from enum import Enum
+
+# This should not be changed in any commit, only used locally
+global_log_disable = False
+
+
+class LogType(Enum):
+    success = 0
+    warning = 1
+    error = 2
+    debug = 3
+
+    def __init__(self, type_id):
+        self._type_strings = {
+            0: 'Success',
+            1: 'Warning',
+            2: 'Error',
+            3: 'Debug'
+        }
+        self._type_id = type_id
+
+    def __str__(self):
+        return self._type_strings[self._type_id]
+
+
+class Logger:
+    """
+    A logger class to be instantiated at the top of each non-test python module for debugging purposes
+    :ivar _filename:
+    :ivar _active:
+    :ivar _firebase: Whether or not to save logs to firebase
+    """
+    _filename = ''
```

Comment:
Since these seem to be instance variables and not class variables, references to them should be removed from this level and kept in __init__ -- otherwise, strange ties between different logger instances could occur.